ra_aid/llm.py

```
@@ -1,93 +1,198 @@
-        ValueError: If the provider is not supported
-    """
-    if provider == "openai":
-        return ChatOpenAI(
-            api_key=os.getenv("OPENAI_API_KEY"),
+
+def create_deepseek_client(
+    model_name: str,
+    api_key: str,
+    base_url: str,
+    temperature: Optional[float] = None,
+    is_expert: bool = False,
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)